lib/logflare/source/bigquery/schema.ex

```diff
@@ -241,7 +242,7 @@ defmodule Logflare.Source.BigQuery.Schema do
   defp next_update() do
     updates_per_minute =
       case Application.get_env(:logflare, :env) do
-        :test -> 600
+        :test -> 60000
         _ -> 6
       end
```

Comment:
Why not set this value in configuration instead of choosing value depending on the environment.

Reply:
moved to config